addons/mail/wizard/mail_activity_schedule.py

```diff
@@ -337,11 +337,12 @@ def action_schedule_plan(self):
                 date_deadline = template._get_date_deadline(self.plan_date)
                 record.activity_schedule(
                     activity_type_id=template.activity_type_id.id,
+                    activity_plan_id=self.plan_id.id,
```

Comment:
Makes more sense to respect activity template (should lead to same result but hey)
```suggestion
                    activity_plan_id=template.plan_id.id,
```